administrator/components/com_patchtester/script.php

```diff
@@ -48,6 +51,24 @@ public function __construct()
 	 */
 	public function postflight($type, $parent)
 	{
-		$this->removeFiles();
+			if ( $type == 'install' ) {
+					$language = JFactory::getLanguage();
+					$language->load('com_patchtester', JPATH_ADMINISTRATOR, null, true);
+					$language->load('com_patchtester', JPATH_SITE, null, true);
+					echo '<h1><img src="../media/com_patchtester/images/icon-48-patchtester.png"> ' . Text::_('COM_PATCHTESTER') . '</h1>';
+					echo '<p>' . Text::_('COM_PATCHTESTER_XML_DESCRIPTION') . '</p>';
+					echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_patchtester">' . Text::_('COM_PATCHTESTER') . '</a></p>';
+					echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_config&view=component&component=com_patchtester#authentication	&path=&return=aHR0cDovL2xvY2FsaG9zdC9wYXRjaDRhMTIvYWRtaW5pc3RyYXRvci9pbmRleC5waHA%2Fb3B0aW9uPWNvbV9wYXRjaHRlc3Rlcg%3D%3D">' . Text::_('COM_PATCHTESTER_OPTIONS') . '</a></p>';
```

Comment:
```suggestion
					echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_config&view=component&component=com_patchtester#authentication&path=&return=aHR0cDovL2xvY2FsaG9zdC9wYXRjaDRhMTIvYWRtaW5pc3RyYXRvci9pbmRleC5waHA%2Fb3B0aW9uPWNvbV9wYXRjaHRlc3Rlcg%3D%3D">' . Text::_('COM_PATCHTESTER_OPTIONS') . '</a></p>';
```
1. Remove tabulator inside the URL of the link's href.
2. Indent all lines from 54 to 72 by one tab less than is is now, to fix code style.

Reply:
Think I've just fixed all these, haven't contributed in a number of years.

Reply:
Indent all lines FROM 54 TO 72, not lines 54 and 72.